EMacco/atm-simulation
# #168778830 Deposit page

#### What does this PR do?
This PR allows a user to be able to deposit money in his account

#### Description of Task to be completed?
- Create field for user to enter amount
- Validate amount
- Display success or error message
- Update balanace
- Display receipt

#### How should this be manually tested?
- Checkout to this branch
- Run `foreman start`
- Open your browser and go to `http://localhost:3000`
- Login
- Enter an amount
- You should see that your account has been credited

#### What are the relevant pivotal tracker stories?
[#168778830](https://www.pivotaltracker.com/story/show/168778830)
